src/split-button.secondary-button.test.miscellaneous.ts

```diff
@@ -0,0 +1,99 @@
+import { html } from 'lit';
+import { expect, test } from './playwright/test.js';
+
+test('defines itself', { tag: '@miscellaneous' }, async ({ mount, page }) => {
+  await mount(
+    () =>
+      html`<glide-core-split-button-secondary-button label="Label">
+        <glide-core-option label="Label"></glide-core-option>
+      </glide-core-split-button-secondary-button>`,
+  );
+
+  const host = page.locator('glide-core-split-button-secondary-button');
+
+  await expect(host).toBeInTheCustomElementRegistry(
+    'glide-core-split-button-secondary-button',
+  );
+});
+
+test(
+  'focuses itself when `focus()` is called',
```

Comment:
```suggestion
  'can be focused programmatically',
```

To match other tests like it.